src/components/implementation/fork_tests/malloc_fork/malloc_fork.c

```diff
@@ -0,0 +1,30 @@
+#include <cos_component.h>
+#include <print.h>
+#include <quarantine.h>
+#include <test_malloc_comp.h>
+
+#define ITER (1024)
+u64_t meas[ITER];
+
+/*
+ * Warning - test assumes Composite can run at least ITER components.
+ * (And this may not take into account other components for this test)
+ */
+	
+void cos_init(void)
+{
+	spdid_t f;
+	printc("Starting a fork test with dynamic memory allocation\n");
+
+	int comp2fork = 12; 	/* this is the HARDCODED spdid for the component we are going to fork. If you change the runscript, change this too. For this test, this is po.o which is an alias for malloccomp.o */
```

Comment:
I wonder if these should be changed to use `cos_init_args`.  There should be a runscript that tests passing in initial arguments to a component that you can see as an example of how to pass arguments from runscript to component.